packages/lib-instrument/test/util/verifier.ts

```diff
@@ -1,19 +1,63 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const {classes} = require('istanbul-lib-coverage');
+import {InstrumenterOptions} from "../../lib/instrumenter";
+
 import {assert} from "chai";
-import clone from "clone";
 import { Instrumenter } from "../../src/instrumenter";
-import { readInitialCoverage } from "../../src/read-coverage";
 
-const FileCoverage = classes.FileCoverage;
 const AsyncFunction = Object.getPrototypeOf(async () => {}).constructor;
 
+export const MOCK_COVERAGE_VARIABLE = "_$COV";
+
+type CoverageType = 'functions' | 'lines' | 'statements' | 'branches';
+type CoveragePerLineAndType = Record<CoverageType, Record<string, number>>;
+
+export const MOCK_VACCINE = `
+var UNIVERSE;
+try {
+  UNIVERSE = (function(){return this;})();
+} catch(e) {
+  UNIVERSE = window || global;
+}
```

Comment:
Would this work as well?

```suggestion
var UNIVERSE = globalThis;
```

Reply:
Good point. Yes, this should also work.